reporting/src/request/page-store.js

```diff
@@ -234,6 +240,24 @@ export default class PageStore {
     this.#pages.set(tabId, nextPage);
   };
 
+  #onNavigationErrorOccured = (details) => {
+    const { frameId, tabId, url, error } = details;
+
+    if (frameId !== 0) {
+      return;
+    }
+
+    // navigation aborted most likely due to redirect
+    // revert to previous page as soon a new navigation will start
+    if (error === 'Error code 2152398850') {
```

Comment:
Would be good to leave some context. Especially, that it is for Firefox only. From what I see, the magic number comes from here:
https://searchfox.org/mozilla-central/source/tools/ts/config/error_list.json#48

```

  "NS_SUCCESS_EVENT_CONSUMED": 4784129,
  "NS_BINDING_SUCCEEDED": 0,
  "NS_BINDING_FAILED": 2152398849,
  "NS_BINDING_ABORTED": 2152398850,  // <---
  "NS_BINDING_REDIRECTED": 2152398851,
  "NS_BINDING_RETARGETED": 2152398852,
  "NS_ERROR_MALFORMED_URI": 2152398858,
  "NS_ERROR_IN_PROGRESS": 2152398863,
  ```